gremlin-go/wait-for-server.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+set -e
+
+host="$1"; shift
+port="$1"; shift
+
+until curl "$host:$port?gremlin=100-1"; do
+  >&2 echo "Gremlin server is unavailable - sleeping"
```

Comment:
not sure if this is an issue, but should there be a limit to how many times it tries? could this wait infinitely and hang if there is a problem?

Reply:
Yes, it would hang is there was a problem with the server, I will add a limit/guard for that.

Reply:
Added a max limit field with usage in the script.